SCSS.md

```diff
@@ -0,0 +1,1356 @@
+# SCSS Linter Rules
+
+###### Below is a list of rules we use for our `scss-lint`.
+###### this is adapted from SCSS-Lint documentation [here](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessarymantissa)
```

Comment:
Can we set the link to the base `README.md` instead of `README.md#unnecessarymantissa`?

Reply:
We can ... 😀